src/compo/viirs_aod2ioda.py

```diff
@@ -311,7 +310,10 @@ def read(self):
                 self.qcfs = self.qcfs[mask_thin]
                 self.obs_time = self.obs_time[mask_thin]
 
-            #  Write out data
+            # create the final mask for output based on time window
```

Comment:
@mer-a-o did you want to just put a suggestion in with the language you would use ;0) do not think @weiwilliam would mind and I'm not comfortable doing it myself at the moment